### PR TITLE
[OSCI] Added new author arnav-mehra

### PR DESCRIPTION
**Pull request by arnav-mehra**
### Description
Adds arnav-mehra as author.
 
### Issues Resolved
None

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/arnav-mehra.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Arnav Mehra
+short_name: Arnav
+twitter:
```
Comment (cwperks) [inline _authors/arnav-mehra.markdown:4]: Please remove this line if its not applicable.

Otherwise this LGTM!

**Review by cwperks — CHANGES_REQUESTED** [review]

File: _authors/arnav-mehra.markdown (hunk 1 above)
Comment (cwperks) [reply _authors/arnav-mehra.markdown:4]: Please also move this to the `_authors/osci_2023/` folder.

Comment (arnav-mehra) [reply _authors/arnav-mehra.markdown]: Thanks for the feedback, Craig! I have made the requested changes.

**Review by dtaivpp — APPROVED** [review]
approved!

**Pull request merged**